lib/miq_expression.rb

```diff
@@ -392,7 +392,7 @@ def self.get_cols_from_expression(exp, options = {})
       elsif exp.key?("count")
         result[exp["count"]] = get_col_info(exp["count"], options)
       elsif exp.key?("tag")
-        # ignore
+      # ignore
```

Comment:
The original was actually better.  Rubocop complains but might not if you put a nil here..

```ruby
elsif exp.key?("tag")
  nil # ignore
else
```